joomla/40-backend-template
# Replace bootstrap-combobox with Awesomplete

Pull Request for Issue #48 
### Summary of Changes

Replace the old bootstrap-combox with Awesomplete.js
### Testing Instructions

Go to `modules/mod_login/mod_login.xml` and add the following parameter:

```
<field name="myeditbox" type="combo" label="combo">
    <option value="1">Option 1</option>
    <option value="2">Option 2</option>
    <option value="3">Option 3</option>
</field>
```

Then open the Login module in the Joomla backend and start typing `Option` in the input field. It should display a list of results.
### Documentation Changes Required

None
